catroid/src/main/java/org/catrobat/catroid/ui/recyclerview/dialog/ReplaceExistingProjectDialogFragment.java

```diff
@@ -122,20 +117,30 @@ public void onPositiveButtonClick(DialogInterface dialog, String textInput) {
 				.setNegativeButton(R.string.notification_download_project_cancel, null)
 				.create();
 
-		radioGroup.setOnCheckedChangeListener(new RadioGroup.OnCheckedChangeListener() {
-			@Override
-			public void onCheckedChanged(RadioGroup group, @IdRes int checkedId) {
-				switch (checkedId) {
-					case R.id.replace:
-						inputLayout.setVisibility(TextView.GONE);
-						alertDialog.getButton(AlertDialog.BUTTON_POSITIVE).setEnabled(true);
-						break;
-					case R.id.rename:
-						inputLayout.setVisibility(TextView.VISIBLE);
-						alertDialog.getButton(AlertDialog.BUTTON_POSITIVE)
-								.setEnabled(textWatcher.validateInput(inputLayout.getEditText().toString(), getContext()) == null);
```

Comment:
Vice verca show the keyboard again here with: 
`ViewUtils.showKeyboard(inputLayout.getEditText());`